### PR TITLE
DSP-671 Update dsp-api rc.16

### PR DESCRIPTION
**Pull request by kilchenmann**
Resolves [DSP-671](https://dasch.myjetbrains.com/youtrack/issue/DSP-671)

### CODE REVIEW

**Review by tobiasschweizer — COMMENTED** [review]
Are there obsolete test data files that can be deleted?

The easiest way to find out is calling `make delete-test-data` before integrating the files from the generated test data.

**Comment by kilchenmann** [comment]
> The easiest way to find out is calling make delete-test-data before integrating the files from the generated test data.

This is what I did.

**Comment by tobiasschweizer** [comment]
> > The easiest way to find out is calling make delete-test-data before integrating the files from the generated test data.
> 
> This is what I did.

ok, I will try that in the afternoon.

File: src/api/admin/permissions/permissions-endpoint-admin.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export class PermissionsEndpointAdmin extends Endpoint {
      */
     getAdministrativePermission(projectIri: string, groupIri: string): Observable<ApiResponseData<AdministrativePermissionResponse> | ApiResponseError> {
```
Comment (tobiasschweizer) [inline src/api/admin/permissions/permissions-endpoint-admin.ts:22]: What about the other possible ways of getting permissions?

Comment (tobiasschweizer) [reply src/api/admin/permissions/permissions-endpoint-admin.ts:22]: see https://github.com/dasch-swiss/knora-api/pull/1708/files

Comment (tobiasschweizer) [reply src/api/admin/permissions/permissions-endpoint-admin.ts:22]: I created a DSP user story: https://dasch.myjetbrains.com/youtrack/issue/DSP-676

**Comment by tobiasschweizer** [comment]
@kilchenmann I will make a manual test.

**Review by tobiasschweizer — APPROVED** [review]
Ok, I tested `getAdministrativePermission` with my local Knora and it worked.

I will create a DSP task for adding e2e tests fro the admin endpoint.

**Pull request merged**